src/cli/__tests__/cli.test.ts

```diff
@@ -71,15 +80,20 @@ describe('visualize', () => {
 });
 
 describe('collect/visualize integration', () => {
+  let htmlFileLocation;
   beforeEach(() => {
     loadProjectStructure('typeScriptIntegrationProject');
+
+    htmlFileLocation = path.join(
+      path.resolve(process.cwd(), '.j1-integration/graph'),
+      'index.html',
+    );
```

Comment:
`resolve` accepts more than two param so you can technically pass in  `index.html`  after `.j1-integration/graph`.

Reply:
Oh, you're right! I didn't notice that in the IntelliSense. I'll fix that now.